internal/service/advancedclustertpf/resource.go

```diff
@@ -101,7 +101,24 @@ func (r *rs) Update(ctx context.Context, req resource.UpdateRequest, resp *resou
 	if diags.HasError() {
 		return
 	}
-	cluster := r.applyClusterChanges(ctx, diags, &state, &plan)
+	stateReq := normalizeFromTFModel(ctx, &state, diags, false)
+	planReq := normalizeFromTFModel(ctx, &plan, diags, false)
+	if diags.HasError() {
+		return
+	}
+	normalizePatchState(stateReq)
+	patchReq, err := update.PatchPayload(stateReq, planReq)
+	if err != nil {
+		diags.AddError("errorPatchPayload", err.Error())
```

Comment:
more human-friendly summary

Reply:
Will handle as part of [next ticket](https://jira.mongodb.org/browse/CLOUDP-286481)